### PR TITLE
Process #10 Styleguide, Prettier, Husky & Yarn

### LINKED ISSUE #13: Create pre push git hook with husky and prettier

### PR DESCRIPTION
**Pull request by juancarrey**
# Issue
#10  Create styleguide, prettier and husky Closes #13 #10

## Description
Added husky, prettier, styleguide and yarn

## Changes Summary
- [x] Added yarn to project, removed package-lock.json and added yarn.lock instead
- [x] Added prettier & husky precommit hook
- [x] Reformatted code
- [x] Added styleguide.md file with basic rules

### CODE REVIEW

**Comment by javierlopezdeancos** [comment]
if you write in the PR 

Closes #13 #10 

when merge and close the PR this issues will be closed automatically

File: package.json [hunk 1]
```diff
@@ -31,13 +31,20 @@
   "homepage": "https://github.com/Devnilson/fisima#readme",
   "devDependencies": {
     "@types/jest": "^25.2.3",
+    "husky": "^4.2.5",
     "jest": "^26.0.1",
-    "prettier": "^2.0.5",
+    "prettier": "2.0.5",
+    "pretty-quick": "^2.0.1",
```
Comment (javierlopezdeancos) [inline package.json:37]: what its that?

**Review by javierlopezdeancos — COMMENTED** [review]
I think that its better add dependencies with their functionality, for example add husky when you implement a hook using husky

File: package.json (hunk 1 above)
Comment (juancarrey) [reply package.json:37]: running prettier quickly on staged files so it runs on pre-commit using husky

**Pull request merged**